runtime-config-linux.md

```diff
@@ -130,6 +130,13 @@ Next parameters can be specified:
 Also known as cgroups, they are used to restrict resource usage for a container and handle
 device access.  cgroups provide controls to restrict cpu, memory, IO, and network for
 the container. For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
+The path to the cgroups can be specified in the Spec. The path is expected to be relative
+to the cgroups mount point. The Spec does not support split hierarchy. The cgroups will be 
+created if they don't exist. 
+
+```json
+   "cgroupsPath": "/myRuntime/myContainer"
```

Comment:
On Wed, Sep 02, 2015 at 05:09:05PM -0700, Vish Kannan wrote:
> Won't this break with [unified
> hierarchy](https://www.kernel.org/doc/Documentation/cgroups/unified-hierarchy.txt)?

Ah, that's interesting reading.  In that case, a single path is fine.

Can we link to those Kernel docs from our “The Spec does not support
split hierarchy.”?